### PR TITLE
Added Merge Queue integration to the workflow

### PR DESCRIPTION
**Pull request by matiasua**
**-  A new condition has been added to trigger the workflow when multiple branches are merged in a merge group.**
**- A confirmation step has been added to ensure that the integration and Docker image building have been successful. It simply ends the workflow with an exit code of 0.**
**- In the branch protection rules, specifically for eol-release/koa and openuchile-release/koa, they were configured to require merge queue using the 'merge commit' merge method and to only merge pull requests that do not fail.**

In summary, this workflow will execute when multiple branches are merged in a merge group or when changes are made to release branches (eol-release/** and openuchile-release/) affecting the files 'Dockerfile', 'requirements/*.txt', and 'themes/'.

### CODE REVIEW

**Comment by fsologureng** [comment]
Me parece excelente. Gracias :)
Puedes hacer un squash en la rama please?

**Comment by fsologureng** [comment]
Estaba pensando que es posible que el workflow `variables` falle siempre en las ramas de la MQ porque no tengo claro si tiene a la vista los datos del evento pr. Veremos si pasa al intentar usarla para este pr.

**Comment by matiasua** [comment]
> Me parece excelente. Gracias :) Puedes hacer un squash en la rama please?

Done;
